pkg/collector/corechecks/cluster/helm/helm.go

```diff
@@ -52,18 +54,23 @@ const (
 // https://helm.sh/docs/faq/changes_since_helm2/#secrets-as-the-default-storage-driver
 type HelmCheck struct {
 	core.CheckBase
-	secretLister      v1.SecretLister
-	configmapLister   v1.ConfigMapLister
+	releases          map[helmStorage]map[string]*release
+	releasesMut       sync.Mutex
```

Comment:
just put the full name here for clarity
```suggestion
	releasesMutex       sync.Mutex
```